lib/open_project/patches/active_record_i18n.rb

```diff
@@ -32,6 +34,15 @@ class Base
 
     def self.human_attribute_name(attr, options = {})
       attr = attr.to_s.delete_suffix("_id")
+
+      options[:default] ||= I18n.t("activerecord.models.#{attr}", count: 1, default: nil)
```

Comment:
To save a lookup, the i18n key can be passed to default as a symbol, and because it's a symbol it will call `I18n.t, on it:

```suggestion
      options[:default] ||= :"activerecord.models.#{attr}"
```